packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -283,79 +535,95 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/crd
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/requirements.yaml packages/rancher-monitoring/charts/requirements.yaml
 --- packages/rancher-monitoring/charts-original/requirements.yaml
 +++ packages/rancher-monitoring/charts/requirements.yaml
-@@ -14,3 +14,98 @@
-     version: "5.3.*"
+@@ -1,16 +1,111 @@
+ dependencies:
+ 
+   - name: kube-state-metrics
+-    version: "2.8.*"
++    version: "2.8.14"
```

Comment:
Should this be hard-coded to a specific version? Not sure if this has been discussed elsewhere, but curious.